pkgs/by-name/sl/slither-analyzer/package.nix

```diff
@@ -1,45 +1,40 @@
 {
+  slither-analyzer,
   lib,
-  nix-update-script,
-  buildPythonPackage,
-  crytic-compile,
-  fetchFromGitHub,
   makeWrapper,
-  packaging,
-  prettytable,
-  pythonOlder,
-  setuptools-scm,
+  fetchFromGitHub,
+  python3Packages,
+  nix-update-script,
   solc,
-  web3,
   withSolc ? false,
   testers,
-  slither-analyzer,
 }:
 
-buildPythonPackage rec {
+python3Packages.buildPythonApplication rec {
```

Comment:
Why this become a python app?

Reply:
Yeah, looks like it should remain in `python-modules` with a `toPythonApplication` in `all-packages.nix`

https://github.com/search?q=lang%3Apython%20%2Fimport.%2Bslither%2F&type=code

Reply:
- slither-analyzer is an app, and I thought it should be moved to the pkgs/by-name.
- and I think had to use `buildPythonApplication`, and I copied some code snippets from some where, see:
  ```
  $ git grep buildPythonApplication pkgs/by-name/
  ```

Reply:
If its an app, maybe it should have a meta.mainProgram ?

Reply:
okay, let me add mainProgram meta.

Reply:
This is already there, this is why I marked my comment as resolved.

Reply:
oh, actually it already has the meta.mainProgram.